### PR TITLE
Fix CSI and add multi mount

### PR DESCRIPTION
**Pull request by ssz1997**
1. Use default /mnt/alluxio/fuse on host machine for alluxio mount point instead of staging path provided by CSI, which for unknown reason doesn't work.
2. Add multi mount check for fuse template

### CODE REVIEW

File: csi/alluxio/controllerserver.go [hunk 1]
```diff
@@ -36,11 +36,6 @@ type controllerServer struct {
 func (cs *controllerServer) CreateVolume(ctx context.Context, req *csi.CreateVolumeRequest) (*csi.CreateVolumeResponse, error) {
 	volumeID := sanitizeVolumeID(req.GetName())
 
-	if err := cs.Driver.ValidateControllerServiceRequest(csi.ControllerServiceCapability_RPC_CREATE_DELETE_VOLUME); err != nil {
```
Comment (ssz1997) [inline csi/alluxio/controllerserver.go:39]: This check is not needed

File: deploy/charts/alluxio/templates/csi/pvc.yaml [hunk 2]
```diff
@@ -16,8 +16,6 @@ kind: PersistentVolumeClaim
 metadata:
   name: {{ $fullName }}-alluxio-csi-fuse-pvc
 spec:
-  accessModes:
-    - ReadWriteOnce
```
Comment (ssz1997) [inline deploy/charts/alluxio/templates/csi/pvc.yaml:20]: Lift the restriction

File: deploy/charts/alluxio/templates/fuse/daemonset.yaml [hunk 3]
```diff
@@ -119,9 +119,6 @@ spec:
             - until nslookup {{ $fullName }}-master-0;
               do sleep 2;
               done
-          volumeMounts:
```
Comment (ssz1997) [inline deploy/charts/alluxio/templates/fuse/daemonset.yaml:122]: Unnecessary volumeMount.

**Review by Kai-Zhang — APPROVED** [review]

**Comment by jayxie8** [comment]
Ask a question
Why Staging Path Provided by CSI, which doesn't work.

**Comment by ssz1997** [comment]
@jayxie8 The initContainer of the fuse pod would try to create the staging path if it doesn't exist, but then the actual container is not able to find that path.

**Pull request merged**